ui/widgets/nuxeo-dialog.js

```diff
@@ -118,6 +118,21 @@ IronOverlayManager._overlayWithBackdrop = function() {
       }
     }
 
+    _clear() {
+      if (this._instance) {
+        const c$ = this._instance.children;
+        if (c$ && c$.length) {
```

Comment:
Is there a reason for using such a particularly unusual variable name such as `c$`?

Reply:
For me, I reused code from `nuxeo-filter` (see [here](https://github.com/nuxeo/nuxeo-elements/blob/maintenance-3.0.x/ui/nuxeo-filter.js#L297-L310)) 😬

Reply:
This was copied from polymer's own code. I don't know the original one but I could find dom-if, which is very similar: https://github.com/Polymer/tools/blob/master/packages/modulizer/fixtures/packages/polymer/source/lib/elements/dom-if.html#L226-L235

Reply:
I'd be fine with renaming it, if you find it easier to read.

Reply:
I don't see any reason to change it, but we can 👍